### ISSUE TITLE
npx tailwindcss --input C:\Users\ayusha\AppData\Local\Temp\scratch_space140e7086\web\styles.tw.css --output C:/'[BUILDER] [ERROR] Failed

### ISSUE THREAD

**Issue by shiwam77** [schultek/jaspr#286]
[BUILDER] [ERROR] ProcessException: The system cannot find the file specified.
[BUILDER] [ERROR]
[BUILDER] [ERROR] Command: npx tailwindcss --input C:\Users\ayusha\AppData\Local\Temp\scratch_space140e7086\web\styles.tw.css --output C:/'[BUILDER] [ERROR] Failed after 821ms
[CLI] [ERROR] Failed building web assets. There is probably more output above.

Step to produce
1.Install Tailwind with the help of  the following command
 ```npm install -g tailwindcss```
 
2. dev_dependencies:
  ``` jaspr_tailwind: ^0.1.1```
   
3.styles.tw.css file in your project's web folder with the following content
   
  ```
     @tailwind base;
     @tailwind components;
     @tailwind utilities;
```
 
4.index.html 
``` <link rel="stylesheet" href="styles.css">```

**Comment by WilliamKarolDiCioccio** [comment]
I'm having the exact same problem

**Comment by schultek** [comment]
Hi. Please use the newest version of jaspr_tailwind and make sure to checkout the changelog, because it no longer uses npx.

I won't be able to fix issues in 0.1.1.

**Comment by shiwam77** [comment]
After installing the tailwind using this command tailwindcss -h its working 
Output:
C:\Users\ayusha>tailwindcss -h

tailwindcss v3.4.10

Usage:
   tailwindcss [--input input.css] [--output output.css] [--watch] [options...]
   tailwindcss init [--full] [--postcss] [options...]


index.html
        <link rel="stylesheet" href="styles.css"/>

File architecture:
web < styles.tw.css
@tailwind base;
@tailwind components;
@tailwind utilities;

Where i am doing wrong here ??
![image](https://github.com/user-attachments/assets/8a1e35af-600c-4b43-8d0a-e67e1946a4c8)


This is my pubspec.yaml

```name: student_management_system
description: A new jaspr project.
version: 0.0.1

environment:
  sdk: '>=3.0.0 <4.0.0'

dependencies:
  flutter:
     sdk: flutter
  jaspr: ^0.15.0
  jaspr_flutter_embed: ^0.3.4
  jaspr_router: ^0.5.1

dev_dependencies:
  build_runner: ^2.4.0
  jaspr_web_compilers: ^4.0.10
  jaspr_builder: ^0.15.0
  flutter_test:
    sdk: flutter
  flutter_lints: ^3.0.0
  jaspr_tailwind: ^0.3.0

jaspr:
  mode: client

flutter:
  uses-material-design: true```
